lib/util/Components.js

```diff
@@ -200,9 +200,22 @@ function componentRule(rule, context) {
     },
 
     /**
+     * Checks to see if our component extends React.PureComponent
+     *
+     * @param {ASTNode} node The AST node being checked.
+     * @returns {Boolean} True if node extends React.PureComponent, false if not
+     */
+    isPureComponent: function (node) {
+      if (node.superClass) {
+        return new RegExp('^(' + pragma + '\\.)?PureComponent$').test(sourceCode.getText(node.superClass));
+      }
+      return false;
+    },
+
+    /**
      * Check if the node is returning JSX
      *
-     * @param {ASTNode} ASTnode The AST node being checked
+     * @param {ASTNode} node The AST node being checked
```

Comment:
Check. And that's not possible because node is reassigned afterwards. Changed back tot original value. Thanks.